lib/govuk_index/presenters/details_presenter.rb

```diff
@@ -1,54 +1,42 @@
 module GovukIndex
   class DetailsPresenter
-    DEFAULT_FIELDS = %w(body parts).freeze
-    FIELDS_BY_FORMAT = {
-      'licence'     => %w(licence_short_description licence_overview),
-      'transaction' => %w(introductory_paragraph more_information)
-    }.freeze
+    extend MethodBuilder
 
-    def initialize(details:, format:, sanitiser:)
+    set_payload_method :details
+
+    delegate_to_payload :licence_identifier
+    delegate_to_payload :licence_short_description
+
+    def initialize(details:, indexable_content_keys:, sanitiser:)
```

Comment:
we can remove format as public methiod from DocTypeInferer as it is no longer used

Reply:
This was incorrect the public method is required